recipe/recipe.yaml

```
@@ -0,0 +1,49 @@
+schema_version: 1
+
+context:
+  name: miniopy-async
+  version: 1.21.3
+  python_min: 3.10
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name | replace('-', '_') }}-${{ version }}.tar.gz
+  sha256: 099c70d136ee43ca680ea628fced0318718254e77751ad3628143d773e9604bb
+
+build:
+  number: 1
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python 3.10.*
```

```suggestion
    - python ${{ python_min }}.*
```

Bug fix for https://github.com/conda-incubator/conda-recipe-manager/issues/307